src/Credentials/GCECredentials.php

```diff
@@ -500,6 +526,56 @@ public function getProjectId(callable $httpHandler = null)
         return $this->projectId;
     }
 
+    /**
+     * Fetch the default universe domain from the metadata server.
+     *
+     * Returns null if called outside GCE.
+     *
+     * @param callable $httpHandler Callback which delivers psr7 request
+     * @return string
+     */
+    public function getUniverseDomain(callable $httpHandler = null): string
+    {
+        if ($this->universeDomain) {
```

Comment:
This should be strict
```suggestion
        if (null !== $this->universeDomain) {
```